src/aggregate/expr/expression.c

```diff
@@ -19,21 +19,21 @@ static void setReferenceValue(RSValue *dst, RSValue *src) {
   RSValue_MakeReference(dst, src);
 }
 
-extern int func_exists(ExprEval *ctx, RSValue *argv, size_t argc, RSValue *result);
-extern int func_case(ExprEval *ctx, RSValue *argv, size_t argc, RSValue *result);
+extern int func_exists(ExprEval *ctx, RSValue **argv, size_t argc, RSValue *result);
+extern int func_case(ExprEval *ctx, RSValue **argv, size_t argc, RSValue *result);
 
 static int evalFuncCase(ExprEval *eval, const RSFunctionExpr *f, RSValue *result) {
   // Evaluate the condition
-  RSValue condVal = RSValue_Undefined();
-  int rc = evalInternal(eval, f->args->args[0], &condVal);
+  RSValue *condVal = RSValue_NewUndefined();
```

Comment:
Do we need to make a new undefined value? Can't we pass a `NULL` pointer to `evalInternal` and update its logic to handle that case?